server-jre/server-jre.nuspec

```diff
@@ -35,7 +35,7 @@ These parameters can be passed to the installer with the use of `--packageParame
 don't directly affect the virtual package, we need to add `--paramsglobal` to pass them to the target/dependent server-jre8.
 
 For example: `choco install server-jre --paramsglobal --packageParameters '"/InstallationPath:C:\tools\Java /Force /Machine"'`.
-</description>
+    </description>
     <releaseNotes>http://www.oracle.com/technetwork/java/javase/documentation/8u-relnotes-2225394.html</releaseNotes >
```

Comment:
The related release note for 8.0.192 is at `https://www.oracle.com/technetwork/java/javase/8u192-relnotes-4479409.html`.